backends/google/drive/google-drive.js

```diff
@@ -245,23 +249,14 @@ export default class GoogleDrive extends Google {
 	 * @returns URL, filename, [folder URL], [folder id], fields.
 	 */
 	static parseURL (source) {
-		let ret = Object.assign(super.parseURL(source), {
+		let ret = Object.assign({
 			filename: undefined,
 			folder: undefined,
 			fields: undefined
-		});
+		}, super.parseURL(source));
```

Comment:
Btw probably not in this PR but I suspect we could clean up the code around defaults too. Right now we have properties that start as `undefined` (like here), then they are overridden by `super.parseURL()`, then we check them again and if they’re empty we apply defaults (and instead of centralizing the code that applies defaults, it's done ad hoc in each class).

Reply:
Yeah. That would be great.